main.js

```diff
@@ -314,21 +323,29 @@ function newGame() {
 //   currentMax.innerText = 100;
 // }
 
+
+function setTimeStart() {
+  gameStart = performance.now();
+}
+
+function setTimeEnd() {
+  gameEnd = performance.now();
+}
+
 function removeWinCard(event) {
   var clickedCard = null;
   if (event.target.classList == 'deleteWinCard') {
     clickedCard = event.target.closest('.game-card');
-    console.log(clickedCard);
     clickedCard.parentNode.removeChild(clickedCard);
   }
 }
 
-function setTimeStart() {
-  gameStart = performance.now();
-  console.log(gameStart);
-}
-
-function setTimeEnd() {
-  gameEnd = performance.now();
-  console.log(gameEnd);
+function clearWinCards(event) {
+  var gameCards = document.querySelectorAll('.game-card');
+  if (event.target.id === 'clearWinCardBtn') {
+    for (var i = 0; i < gameCards.length; i++) {
+      gameCards[i].remove();
+    }
+    clearWinCardBtn.classList.remove('active');
+  }
```

Comment:
We talked about using event as a parameter a little bit, but I'm still a bit fuzzy on it. I'm excited for the event bubbling lessons we have this week - hopefully that will give us more clarity on when it works and when it doesn't!